client/src/pages/Home/Footer/Footer.js

```diff
@@ -0,0 +1,79 @@
+import React, { useState } from "react";
+import "./Footer.css";
+import { FacebookOption, Twitter, Instagram } from "grommet-icons";
+
+export function Footer() {
+	const [isShown, setIsShown] = useState(false);
+	return (
+		<div id="footer">
+			{/* Section11 */}
+			<div className="home_section11_parent">
+				<div className="home_section11_child1">
+					<img
+						className="footer-logo"
+						src="https://i.postimg.cc/VkdMYP4S/Logo.jpg"
+						alt="logo"
+					/>
+				</div>
+				<div className="home_section11_child2">
+					<div>
+						<p>Do want to be part of us?</p>
+					</div>
+					<div>
+							<a
+								href="https://example.com/"
+								target="_blank"
+								rel="noreferrer"
+								className="footer-button"
+								onMouseEnter={() => setIsShown(true)}
```

Comment:
just wonder why you are using onMouseEnter only? do you need yo add onClick? also is href correct

Reply:
The instruction in ticket on Trello is " When the User hovers over the Get Started button, the colour of the button changes, and a text box ''Get Started '' appears just below the button.".
There is no instruction for a function when users click the buttun, so I use only onMouseEnter in this component.